Cargo.toml

```diff
@@ -15,6 +15,7 @@ opt-level = 3
 
 [profile.release]
 opt-level = 3
+debug = true
```

Comment:
is it needed also, in the main branch? in this case, We need to add some section "How to build your simulation"